public/stylesheets/styles.css

```diff
@@ -0,0 +1,121 @@
+.btn-spacing {
+  margin: 5px;
+}
+
+.btn-window {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 15px;
+  width: 15px;
+  border-radius: 50%;
+  font-family: 'Roboto', sans-serif;
+}
+
+.btn-close {
+  background-color: #fc5855;
+}
+
+.btn-min {
+  background-color: #fec141;
+}
+
+.btn-open {
+  background-color: #35cb4b;
+}
+
+// .btn-close:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+// .btn-min:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+// .btn-open:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+.calculator-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #8E8E8E;
+  width: 100%;
+  height: 700px;
+}
+
+.calculator-font {
```

Comment:
The "cascading" part of CSS means that styles apply to the element you set them on as well as their descendants. So you do not need the `. calculator-font` classname because you can just set these styles on the outer calculator element and it will cascade down to all the elements within the whole calculator.